src/shared/sam/debugger/typescriptSamDebug.ts

```diff
@@ -118,6 +118,10 @@ export async function makeTypescriptConfig(config: SamLaunchRequestArgs): Promis
  */
 async function compileTypeScript(config: NodejsDebugConfiguration): Promise<void> {
     if (config.invokeTarget.target === 'code') {
+        const tscResponse = await new ChildProcess(true, 'tsc -v').run()
```

Comment:
always pass args separately / as a list.

```suggestion
        const tscResponse = await new ChildProcess(true, 'tsc', undefined, '-v').run()
```